tagbot/action/repo.py

```diff
@@ -46,14 +56,17 @@ def __init__(
         ssh: bool,
         gpg: bool,
         lookback: int,
+        github_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
```suggestion
        github_kwargs: Optional[Dict[str, object]] = None,
```

More details [here](https://mypy.readthedocs.io/en/latest/dynamic_typing.html#any-vs-object)
Then the import change can also be reverted.

Reply:
Hm, it seems like my earlier comment disappeared.
Having `object` instead of Any causes errors in static analysis, I assume habing `object` implies all values will have same value, which is not case here, which is why I think `Any` is more correct, although it results in annoyingly long imports. 
```
tagbot/action/repo.py:59: error: Argument 4 to "Github" has incompatible type "**Dict[str, object]"; expected "Optional[str]"
tagbot/action/repo.py:59: error: Argument 4 to "Github" has incompatible type "**Dict[str, object]"; expected "int"
tagbot/action/repo.py:59: error: Argument 4 to "Github" has incompatible type "**Dict[str, object]"; expected "str"
tagbot/action/repo.py:59: error: Argument 4 to "Github" has incompatible type "**Dict[str, object]"; expected "bool"
```

Reply:
It's not assuming that the values are all of the same type, it's acknowledging that they can now be of any type. So the mypy errors are technically correct: if you can pass in any keywords you want to `Github`, then you can indeed give it faulty input. I think here we can just [tell mypy to ignore the `Github(...)` line](https://mypy.readthedocs.io/en/stable/common_issues.html#spurious-errors-and-locally-silencing-the-checker) since we assume that you know what you're doing if you use extra keywords.

Reply:
I see, you're right, fixed.